src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -102,22 +133,49 @@ function getAttribute(component, definition) {
 }
 
 /**
- * Get an array of attributes corresponding the definedAttributes array.
- * @param {Object} component - Component containing the available attribute and all definitions.
- * @return {Array} an array of attributes.
+ * Get an array of attributes corresponding to the definedAttributes array.
+ * @param {Object} component - Component containing the referenced attribute and all definitions.
+ * @return {Array} an array of referenced attributes.
  */
-function getAttributes(component) {
+function getReferencedAttributes(component) {
   return component.definition.definedAttributes.map((defAttr) => getAttribute(component, defAttr));
 }
 
+/**
+ * Get an array of attributes corresponding to the attributes array.
+ * @param {Object} component - Component containing the unreferenced attribute.
+ * @return {Array} an array of unreferenced attributes.
+ */
+function getUnreferencedAttributes(component) {
+  return component.attributes
+    .filter((attr) => attr.definition === null);
+}
+
+/**
+ * Get an array of attributes corresponding to the attribute key.
+ * @param {String} key - Attribute key.
+ * @return {Array} an array of attributes.
+ */
+function getSelectedComponentAttributes(key) {
```

Comment:
Why not making a computed ?
```js
const referencedAttributes = computed();
const unreferencedAttributes = computed();
```
?

Reply:
Since I wanted to use the same `input-wrapper` for both `referenced` and `unreferenced` attributes, these values should not be separated. But this will change with the new way of displaying unreferenced attributes (with two separate inputs etc)